src/overlays/actors/ovl_En_Bird/z_en_bird.c

```diff
@@ -47,81 +47,78 @@ void EnBird_Init(Actor* thisx, PlayState* play) {
     SkelAnime_Init(play, &this->skelAnime, &gBirdSkel, &gBirdFlyAnim, NULL, NULL, 0);
     ActorShape_Init(&this->actor.shape, 5500, ActorShadow_DrawCircle, 4);
     this->unk_194 = 0;
-    this->unk_198 = 0;
-    this->unk_1C0 = 0x9C4;
+    this->timer = 0;
+    this->rotYStep = 2500;
     this->actor.colChkInfo.mass = 0;
-    this->unk_1A8 = 1.5f;
-    this->unk_1AC = 0.5f;
-    this->unk_1A0 = 0.0f;
-    this->unk_1A4 = 0.0f;
-    this->unk_1B8 = 0.0f;
-    this->unk_1B0 = 40.0f;
+    this->speedXZTarget = 1.5f;
+    this->speedXZStep = 0.5f;
+    this->posYMag = 0.0f;
+    this->rotYMag = 0.0f;
+    this->posYWaveDelta = 0.0f;
+    this->flightDistance = 40.0f;
     this->unk_1BC = 70.0f;
-    func_809C1CAC(this, this->actor.params);
+    EnBird_Reset(this, this->actor.params);
 }
 
 void EnBird_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_809C1CAC(EnBird* this, s16 params) {
+void EnBird_Reset(EnBird* this, s16 params) {
     f32 frameCount = Animation_GetLastFrame(&gBirdFlyAnim);
     f32 playbackSpeed = this->unk_19C ? 0.0f : 1.0f;
     AnimationHeader* anim = &gBirdFlyAnim;
 
-    this->unk_198 = Rand_S16Offset(5, 0x23);
+    this->timer = Rand_S16Offset(5, 35);
     Animation_Change(&this->skelAnime, anim, playbackSpeed, 0.0f, frameCount, ANIMMODE_LOOP, 0.0f);
-    EnBird_SetupAction(this, func_809C1D60);
+    EnBird_SetupAction(this, EnBird_StopMove);
 }
 
-void func_809C1D60(EnBird* this, PlayState* play) {
-    f32 fVar2 = sinf(this->unk_1B4);
+void EnBird_StopMove(EnBird* this, PlayState* play) {
 
-    this->actor.shape.yOffset = this->actor.shape.yOffset + fVar2 * this->unk_1A0;
+    this->actor.shape.yOffset += sinf(this->posYWave) * this->posYMag;
     Math_SmoothStepToF(&this->actor.speedXZ, 0.0f, 0.1f, 0.5f, 0.0f);
 
     if (this->unk_19C != 0) {
         this->skelAnime.playSpeed = this->actor.speedXZ + this->actor.speedXZ;
```

Comment:
`this->actor.speedXZ + this->actor.speedXZ`
probably matches as
`2.0f * this->actor.speedXZ`
or `this->actor.speedXZ * 2.0f`

also `if (this->unk_19C != 0)` could be `if (this->unk_19C)` :shrug: no big deal